terraform/18f.gov.tf

```diff
@@ -1782,3 +1782,21 @@ resource "aws_route53_record" "d_18f_gov__acme_challenge_c2_18f_gov_cname" {
 output "d_18f_gov_ns" {
   value = aws_route53_zone.d_18f_gov_zone.name_servers
 }
+
+# queues.federalistapp.18f.gov - CNAME -----------------
+resource "aws_route53_record" "d_18f_gov_queues_federalistapp_18f_gov_cname" {
+  zone_id = "${aws_route53_zone.d_18f_gov_zone.zone_id}"
```

Comment:
@apburnes This is probably not a problem but I noticed that your syntax
```
  zone_id = "${aws_route53_zone.d_18f_gov_zone.zone_id}"
```
is different from that used for, say, the `c2.18f.gov` record a few blocks above:
```
  zone_id = aws_route53_zone.d_18f_gov_zone.zone_id
```
If either will work then I won't worry about it, but I wanted to note it.

Reply:
Thanks! Updated for consistency.